BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpViewController.swift

```diff
@@ -0,0 +1,170 @@
+        appNameLabel.textAlignment = .center
+        appNameLabel.textColor = .black
+    }
+    
+    private func addLogoImageView() {
+        view.addSubview(logoImageView)
+        addLogoLabel()
+        
+        logoImageView.image = UIImage(named: "logo")
+    }
+    
+    private func addLogoLabel() {
+        logoImageView.addSubview(logoLabel)
+        
+        logoLabel.text = String(localized: "logoLabelText").uppercased()
+        logoLabel.font = UIFont(name: FontName.interExtraBold.rawValue, size: 15.0)
+        logoLabel.textAlignment = .left
+        logoLabel.textColor = UIColor(hexNumber: 0xFFFFFF)
+    }
+    
+    private func addPhraseLabel() {
+        view.addSubview(phraseLabel)
+        
+        phraseLabel.text = String(localized: "phraseLabelText").uppercased()
+        phraseLabel.font = UIFont(name: FontName.SFProDisplayMedium.rawValue, size: 14.0)
+        phraseLabel.textAlignment = .center
+        phraseLabel.textColor = .black
+        phraseLabel.lineBreakMode = .byWordWrapping
+        phraseLabel.numberOfLines = 0
+    }
+    
+    private func addStartButton() {
+        view.addSubview(startButton)
+        
+        var buttonConfiguration = UIButton.Configuration.filled()
+        buttonConfiguration.title = String(localized: "startButtonTitle").uppercased()
+        buttonConfiguration.cornerStyle = .large
+        buttonConfiguration.baseBackgroundColor = UIColor(hexNumber: 0x090909)
+
+        startButton.configuration = buttonConfiguration
+        
+        startButton.addTarget(self, action: #selector(startButtonTapped), for: .touchUpInside)
+    }
+    
+    private func setupViewConstraints() {
+        let safeArea = view.safeAreaLayoutGuide
+        
+        backgroundImageView.snp.makeConstraints { make in
+            let heightMultiplier: CGFloat = 445 / 844
+
+            make.height.equalToSuperview().multipliedBy(heightMultiplier)
+            make.bottom.equalToSuperview()
+            make.leading.trailing.equalTo(safeArea)
+        }
+        
+        welcomeLabel.snp.makeConstraints { make in
+            make.top.equalTo(safeArea).inset(35)
+            make.centerX.equalTo(safeArea)
+        }
+        
+        appNameLabel.snp.makeConstraints { make in
+            make.top.equalTo(welcomeLabel.snp.bottom)
+            make.centerX.equalTo(safeArea)
+        }
+        
+        logoImageView.snp.makeConstraints { make in
+            let multiplayer: CGFloat = 196 / 390
+            
+            make.centerX.equalTo(safeArea)
+            make.centerY.equalTo(safeArea).offset(-35)
+            make.size.equalTo(safeArea.snp.width).multipliedBy(multiplayer)
+        }
+        
+        logoLabel.snp.makeConstraints { make in
+            make.centerX.centerY.equalTo(logoImageView)
+        }
+        
+        phraseLabel.snp.makeConstraints { make in
+            make.bottom.equalTo(startButton.snp.top).inset(-30)
```

Comment:
let's use only positive numbers in constraints. There is inset and offset, I suppose for you needs offset is fine

Reply:
Constraints changed